### ISSUE TITLE
SVG pan does not work on mobile

### ISSUE THREAD

**Issue by vincerubinetti** [anvaka/panzoom#120]
Test the following link on iOS or Android:
https://jsfiddle.net/uwxcmbyg/609/show

Trying to pan the image results in it shooting off the top left of the screen.

Pinch-to-zoom seems to work fine. 

Also, the regular dom pan example seems to work fine on mobile:
https://anvaka.github.io/panzoom/demo/dom.html

**Comment by Rendrik** [comment]
Tinkering with it this morning, it seems there doesn't need to be a special transform for SVG, as replacing the transform to screen function to just set x/y appeared to fix it.
```
function transformToScreen(x, y) {
  storedCTMResult.x = x
  storedCTMResult.y = y

  return storedCTMResult
}
```
I validated this natively on my android device with Chrome. Not sure about other mobile browsers.

**Comment by hope55** [comment]
Same issue here. I tried Rendrik 's solution, it works perfect.  @Rendrik, I've seen you updated your fork, would you send a pull request to the original repo? I hope the fix will be merged in the next release.

**Comment by moonbear79** [comment]
> Tinkering with it this morning, it seems there doesn't need to be a special transform for SVG, as replacing the transform to screen function to just set x/y appeared to fix it.
> 
> ```
> function transformToScreen(x, y) {
>   storedCTMResult.x = x
>   storedCTMResult.y = y
> 
>   return storedCTMResult
> }
> ```
> 
> I validated this natively on my android device with Chrome. Not sure about other mobile browsers.

@Rendrik revealed the right way, but... utility functions that invoke `transformToScreen(x, y)` as `zoomAbs()` etc. stops working properly.
Solution is to add function with new name along with original
```
function transformToScreenOnTouch(x, y) {
  storedCTMResult.x = x;
  storedCTMResult.y = y;
  return storedCTMResult;
}
```
and call it in `handleTouchMove(e)`.
```
/*var point = transformToScreen(dx, dy);*/
var point = transformToScreenOnTouch(dx, dy);
```
At least that worked for me.

**Comment by lucafaggianelli** [comment]
Same issue here, to use your workaround I must rebuild the lib on my own?

**Comment by anvaka** [comment]
Sorry for the issue, friends.

https://github.com/anvaka/panzoom/commit/49be4a1bd6361598b79f29fe99adc2c125d93678 fixes it. Available starting from `v8.7.3`
